lib/parsers/moov_parser.rb

```diff
@@ -18,7 +18,7 @@ class FormatParser::MOOVParser
   MP4_MIXED_MIME_TYPE = 'video/mp4'
 
   def likely_match?(filename)
-    filename =~ /\.(mov|m4a|ma4|mp4|aac|m4v)$/i
```

Comment:
What happened with `aac`? 🤔

Reply:
Ruben raised [here](https://github.com/WeTransfer/format_parser/pull/222#discussion_r1013725910) that it doesn't really make any sense for it to be here. It's not in the FTYP map and Costin wrote a [dedicated parser](https://github.com/WeTransfer/format_parser/blob/master/lib/parsers/aac_parser.rb) for it a short while ago.

Reply:
Alright, seems reasonable. Let's just make sure we're not altering any of the unseen with this.